spec/features/user_can_login_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.feature "Login", type: :feature do
```

Comment:
I wonder whether it's possible to define helper methods for signing-up and logging-out that you can call, so that this test would just have the code for logging-in? (e.g. `sign_up, log_out, click_link "Login"`, etc. )Otherwise it looks a bit like it's testing all three features (which I understand it kind of has to) until you see the separate tests for the other two. But also keeping tests dry is a matter of taste, so I get if you've intentionally preferred to do it like this!

Reply:
Absolutely agree with this!!